<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlayListItem</class>
 <widget class="QWidget" name="XPlayListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlayListItem</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetBg" native="true">
     <property name="styleSheet">
      <string notr="true">#widgetBg
{
border-bottom:1px solid #c0c0c0;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
min-width:30px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
font-size:18px;
}</string>
        </property>
        <property name="text">
         <string>url</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSet">
        <property name="text">
         <string>+</string>
        </property>
        <property name="icon">
         <iconset resource="XWallpaper2.qrc">
          <normaloff>:/XWallpaper2/res/set.png</normaloff>:/XWallpaper2/res/set.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnStart">
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="icon">
         <iconset resource="XWallpaper2.qrc">
          <normaloff>:/XWallpaper2/res/start.png</normaloff>:/XWallpaper2/res/start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDelete">
        <property name="text">
         <string>X</string>
        </property>
        <property name="icon">
         <iconset resource="XWallpaper2.qrc">
          <normaloff>:/XWallpaper2/res/delete.png</normaloff>:/XWallpaper2/res/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XWallpaper2.qrc"/>
 </resources>
 <connections/>
</ui>
